util/url_value.go

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Are the `Set` and `Type` receivers used anywhere?

Reply:
Set is used to set argument passed to the variable.

Since URLValue is an implementation of Value interface in pflag, all three functions are required - https://godoc.org/github.com/spf13/pflag#Value

Reply:
Thanks. I missed this.